Softeq.XToolkit.Common/Helpers/Hashing/Hash.cs

```diff
@@ -0,0 +1,13 @@
+// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+namespace Softeq.XToolkit.Common.Helpers.Hashing
+{
+    /// <summary>
+    /// Class helps to get a hash code for a number of objects combined
+    /// </summary>
+    public static class Hash
+    {
+        public static IHashCalculator Get() => new HashCalculator();
```

Comment:
`Get()` can be renamed to `Default` and used as property

Reply:
Please look at this [comment](https://github.com/Softeq/XToolkit.WhiteLabel/pull/280#discussion_r401395475)